context.go

```diff
@@ -33,6 +33,7 @@ type Context interface {
 	Redirect(int, string, ...interface{}) error
 	Data() map[string]interface{}
 	Flash() *Flash
+	CurrentRoute() RouteInfo
```

Comment:
Let's try and not add more things to the context unless we really need it. The `current_route` is already in the context data, if people need it in a handler they can get it that way. If it becomes a problem later we can extend the context, but i'm already nervous as to how big it already is.